<?xml version="1.0"?>
<launch>
<arg name="robot_namespace" default="/"/> 
<arg name="verbosity_level" default="NOTICE"/>
<arg name="left_drive_config" default="left_drive.yaml"/>
<arg name="right_drive_config" default="right_drive.yaml"/>
<node name="tankdrive_node" pkg="crawler_app" type="tankdrive_node"  output="screen" clear_params="true">
        <param name="robot_namespace" value="$(arg robot_namespace)"/>
        <param name="startup_delay"       value="0.0"/>
	<param name="verbosity_level"       value="$(arg verbosity_level)"/>
        <param name="require_pps_to_start"  value="false"/>    
        <param name="loop1_rate"            value="1"/> 
        <param name="loop2_rate"            value="0.1"/>
        <param name="loop3_rate"            value="5"/>
        <rosparam command="load"  param="left_drive" file="$(arg left_drive_config)" />
        <rosparam command="load"  param="right_drive" file="$(arg right_drive_config)" />

</node>
</launch>
